src/style.css

```
@@ -0,0 +1,82 @@
+body {
+  margin: 0;
+}
+
+* {
+  box-sizing: border-box;
+}
+
+.block--1 {
+  background-color: red;
+}
+
+.block--2 {
+  background-color: #c00;
+}
+
+.block--3 {
+  background-color: #900;
+}
+
+.block--4 {
+  background-color: #600;
+  order: 2;
+}
+
+.block--5 {
+  background-color: #300;
+  order: 1;
+}
+
+.block--6 {
+  background-color: black;
+}
+
+.container {
+  display: flex;
+  flex-wrap: wrap;
+  align-items: stretch;
+}
+
+.block {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  text-align: center;
+  font-family: Arial, sans-serif;
+  color: white;
+  font-size: 100px;
+  min-width: 300px;
+  height: 300px;
+  flex-basis: 33.3%;
```

```suggestion
  flex-basis: 33.33%;
```
Now there is a visible white line on the right end of the screen